source/platforms/databricks.md

```diff
@@ -63,9 +63,70 @@ gdf
 0   1   4
 1   2   5
 2   3   6
+```
+
+## Multi-node Dask cluster
+
+We now provide a [dask-databricks](https://github.com/jacobtomlinson/dask-databricks) CLI tool (via [`conda`](https://github.com/conda-forge/dask-databricks-feedstock) and [`pip`](https://pypi.org/project/dask-databricks/)) to simplify the Dask cluster startup process within Databricks.
+
+Running `pip install dask-databricks` should launch a dask scheduler in the driver node and workers on remaining nodes in just a few minutes.
+
+### Create init-script
+
+To get started, you must first configure an [initialization script](https://docs.databricks.com/en/init-scripts/index.html) to install `dask`, RAPIDS libraries and all other dependencies for your project.
+
+Databricks recommends using [cluster-scoped](https://docs.databricks.com/en/init-scripts/cluster-scoped.html) init scripts stored in the workspace files.
+
+Navigate to the top-left **Workspace** tab and click on your **Home** directory then select **Add** > **File** from the menu. Create an `init.sh` script with contents:
+
+```bash
+#!/bin/bash
+set -e
+
+# The Databricks Python directory isn't on the path in
+# databricksruntime/gpu-tensorflow:cuda11.8 for some reason
+export PATH="/databricks/python/bin:$PATH"
+
+# Install RAPIDS (cudf & dask-cudf) and dask-databricks
+/databricks/python/bin/pip install --extra-index-url=https://pypi.nvidia.com \
+      cudf-cu11 \  # installs cudf
+      dask[complete] \
+      dask-cudf-cu11  \ #installs dask-cudf
```

Comment:
Sadly we can't add comments after the `\` like this as they get included in the command. My cluster was failing to start until I removed these.

```suggestion
      cudf-cu11 \
      dask[complete] \
      dask-cudf-cu11  \
```